strawberry/utils/pick.py

```diff
@@ -0,0 +1,12 @@
+from typing import Optional, TypeVar
+
+
+T = TypeVar("T")
+
+
+def pick_not_none(*args: Optional[T]) -> T:
```

Comment:
It is used in schema_converter

Reply:
Are you sure? I see neither current reference to `pick_not_none` nor one added in this PR.

Reply:
Oh nvm, I got tricked. GitHub had completely minimized the diff in the PR because it was a large change and I hadn't noticed. That made `ctrl`+`f` stop working too

Reply:
There is a semantic difference between `description_sources is None` (Don't change it) and `description_sources == []` (Don't add any description), which isn't captured by the `a or b`

Reply:
Hmm that's a fair point as it's currently implemented. This is again making me question having the list of sources. If we didn't have a list, having a `DescriptionSource.NONE` (as a false-y value) would make perfect sense.

Reply:
Having a false-y `DescriptionSource.NONE` results in the same problem: It gets ignored and the code fallbacks to the parent/global config.

```python
config=StrawberryConfig(description_sources=DescriptionSources.ALL)

[...]

@strawberry.type(description_sources=DescriptionSource.NONE)
class UndocumentedClass:
  """ Secrets that shouldn't be exposed """
   ...

[...]

description_sources = type.description_sources or config.description_sources  
# NONE or ALL => ALL
# Secrets are exposed!  O.O
```

This would work if NONE was not False-y, but that goes against normal semantics of `enum.Flag` :sweat: 

I'll keep this method for now